qemu/tests/ovs_qos.py

```diff
@@ -131,6 +131,8 @@ def __get_vminfo():
             """
             login_timeout = float(params.get("login_timeout", 360))
             clear_iptables_cmd = "service iptables stop; iptables -F"
+            stop_firewall_cmd = "systemctl stop firewalld||"
+            stop_firewall_cmd += "service firewalld stop"
```

Comment:
"iptalbes -F" are also used to stop firewall.
You can use:
clear_iptables_cmd = "systemctl stop firewalld || service iptables stop || iptables -F"